src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Analyze/ConsoleAnalyzeCommand.cs

```diff
@@ -17,6 +18,11 @@ public ConsoleAnalyzeCommand()
                 Host.CreateDefaultBuilder()
                     .UseConsoleUpgradeAssistant<ConsoleAnalyze>(options, result)
                     .RunUpgradeAssistantAsync(token));
+
+            if (FeatureFlags.IsRequested("FORMAT"))
```

Comment:
Great! Can you add a constant in the FeatureFlags class for AnalyzeFormat (or some name) so you're not using a magic string here?